android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerModule.java

```diff
@@ -70,6 +71,8 @@
   private static final String KEY_PAN_MIN_POINTERS = "minPointers";
   private static final String KEY_PAN_MAX_POINTERS = "maxPointers";
   private static final String KEY_PAN_AVG_TOUCHES = "avgTouches";
+  private static final String KEY_NUMBER_OF_TOUCHES= "numberOfTouches";
```

Comment:
space before `=` here and in the next line

Also we've been using a different naming before – for PanHandler we use `minPointers` so would be good to make it consistent and call this attribute the same way

Reply:
That might be confusing. Because it's not just minimal number of touches, but _exact_ number o them. If you try to fling with more than given number of fingers, it won't work